### PR TITLE
Try to use israeli-bank-scrapers-core from renderer

### LINKED ISSUE #79: Wrong work with IPC
We are working incorrectly with IPC.

For example, we are listening in the `main` process for the event `decrypt`.

Each `Importer.vue` sends a data to `decrypt`, and **register** for the `decrypt-result` for the result.
When one `Importer` sends an object to `decrypt`, **all the `Importer`s** that already registered to `decrypt-result` **will receive the decrypted object** when the `main` process will send it!

### PR DESCRIPTION
**Pull request by baruchiro**
Needs to validate the release output runs in the user machine.

Related to #79, trying to reduce IPC usage.

EDIT: Now will fix #79

### CODE REVIEW

**Comment by Arielgordon123** [comment]
interesting, so the problem only happens if the import from `israeli-bank-scrapers-core` is to the store

File: vue.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ module.exports = {
   pluginOptions: {
     electronBuilder: {
       // List native deps here if they don't work
-      externals: ['keytar'],
+      externals: ['keytar', 'israeli-bank-scrapers-core'],
```
Comment (baruchiro) [inline vue.config.js]: @Arielgordon123 

> interesting, so the problem only happens if the import from `israeli-bank-scrapers-core` is to the store

No, the problem is when calling to `israeli-bank-scrapers-core`, who is calling to `ws`, in the `renderer` process. So the important change is this change, and this is why I said:

> Needs to validate the release output runs in the user machine.

**Comment by baruchiro** [comment]
I remembered the Sandbox feature in Windows, and I checked the artifact created from this PR, and the attached image proves that it works as expected, right?

image

**Comment by baruchiro** [comment]
Do not merge, I'm trying to create a test for that

**Comment by baruchiro** [comment]
@Arielgordon123 Ready

**Pull request merged**